src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -14,22 +14,28 @@ export interface MakeInvoiceArgs {
   memo: string;
 }
 
-export interface MakeInvoiceResponse {
-  data: {
-    paymentRequest: string;
-    rHash: string;
-  };
-}
+export type MakeInvoiceResponse =
```

Comment:
I think our response types have to describe a happy path response. And when an error happens somewhere we should throw / catch it.

Reply:
Addressed in #714